Browser Test Actions and Advanced Settings pages (#6489)
DataDog/documentation: content/en/synthetics/browser_tests/_index.md

---
title: Browser Tests
kind: documentation
description: Simulate and monitor user journeys from specific locations.
aliases:
  - /synthetics/browser_check

>   - /synthetics/browser_test

Reviewer: ```suggestion
  - /synthetics/browser_tests
```

### Alert conditions

An alert is triggered if any assertion fails for `<INSERT_NUMBER>` minutes from any `<INSERT_NUMBER>` of `<NUMBER_OF_CHOSEN>` locations.

### Notifications

To configure your notifications:

1. Enter a **message** for the browser test. This field allows standard [Markdown formatting][6]. Notification messages include the **message** defined in this section and information about which assertion failed and why.
2. Choose your [services][7] and/or team members to notify.
3. Click **Save Details and Record Test** to save your browser test.
4. Start to record your test.

## Record test

Tests can be only recorded from **[Google Chrome][8]**. To record your test, download the [Datadog Record Test extension for Google Chrome][9].

{{< img src="synthetics/browser_tests/browser_check_record_test.png" alt="Browser test record test"  >}}

1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.

> Hitting **Open in a pop-up** and **Shift** will open the pop up in incognito mode. This is useful to start recording your test from a fresh browser free from already logged in sessions, cookies from your existing browser, etc.

Reviewer 2: ```suggestion
Hitting **Open in a pop-up** and **Shift** opens the pop up in incognito mode. This is useful to start recording your test from a fresh browser free from already logged in sessions, cookies from your existing browser, etc.
```

2. Click on **Start recording** to begin recording your browser test.
3. Your actions are recorded and used to create steps within your browser test scenario.
4. Use the actions available in the upper left corner to enrich your scenario:
    {{< img src="synthetics/browser_tests/browser_test_step.png" alt="Browser Test steps"  style="width:80%;">}}

    **Note**: **Your last browser test step must be an Assertion**, otherwise there is nothing to check.

> 5. Once you have finished your Scenario, click on **Save and Launch Test**.

Reviewer 2: ```suggestion
5. Once you have finished your scenario, click on **Save and Launch Test**.
```

## Actions

> After saving a Browser Test you can record [Actions][10] as a series of steps, which you can then edit or build on. Actions can also be configured with [advanced options][11].

Reviewer 2: ```suggestion
After saving a browser test, you can record [Actions][10] as a series of steps, which you can then edit or build on. You can also configure actions with [advanced options][11].
```

> After saving a Browser Test you can record [Actions][10] as a series of steps, which you can then edit or build on. Actions can also be configured with [advance settings][11].

Reviewer 3: ```suggestion
After saving a Browser Test you can record [Actions][10] as a series of steps, which you can then edit or build on. Actions can also be configured with [advanced settings][11].
```

I think?

Reviewer 3: I'm not seeing this sentence, but I'm seeing all the actions on the general browser test page - is it just a cache issue?

Author: Nope, definitely was a me issue - sorry! I ended up pushing the commit without removing from the original file. Will remove this and the other sections we talked about removing.

Author: This is odd. It's not in my file locally or here, but is showing up still on the staging page. @l0k0ms can you take a look into this?

Reviewer: @sarina-dd fixed it, the old file wasn't deleted and the aliases were doing a circular dependency. This should be fixed now :)
